src/script_chainer/services/log_notifier.py

```
@@ -0,0 +1,84 @@
+    def _flush(self) -> None:
+        with self._lock:
+            if len(self._pool) == 0:
+                return
+            items = list(self._pool.items)
+            self._pool.clear()
+        try:
+            self._ctx.push_service.push_merged_async(
+                title=self._title,
+                items=items,
+            )
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)